conf/modules.config

```diff
@@ -519,30 +524,37 @@ process {
         ]
     }
 
+    // Overwrite default SWF tag and prefix
+    withName: ".*BAM_SPLIT_BY_REGION:SAMTOOLS_VIEW" {
+        tag = { "${meta.reference}:${meta.genomic_region}|${meta.sample_id}_${meta.library_id}" }
+        ext.prefix = { "${meta.sample_id}_${meta.library_id}_${meta.reference}_${meta.genomic_region}_dedupped" }
+        publishDir = [
+            enabled: false
+        ]
+    }
+
     withName: ".*BAM_SPLIT_BY_REGION:SAMTOOLS_INDEX" {
-        tag = { "${meta.reference}|${meta.id}" }
-        ext.prefix = { "${meta.id}_${meta.library_id}_${meta.reference}_dedupped" }
+        tag = { "${meta.reference}:${meta.genomic_region}|${meta.sample_id}_${meta.library_id}" }
+        ext.prefix = { "${meta.sample_id}_${meta.library_id}_${meta.reference}_${meta.genomic_region}_dedupped" }
         ext.args = { params.fasta_largeref ? "-c" : "" }
         publishDir = [
-            path: { "${params.outdir}/deduplication/" },
-            mode: params.publish_dir_mode,
-            pattern: '*.bai'
+            enabled: false
         ]
     }
 
-    withName: ".*DEDUPLICATE:SAMTOOLS_SORT" {
-        tag = { "${meta.reference}|${meta.id}" }
-        ext.prefix = { "${meta.id}_${meta.library_id}_${meta.reference}_dedupped" }
+    withName: SAMTOOLS_SORT_DEDUPPED {
+        tag = { "${meta.reference}|${meta.sample_id}_${meta.library_id}" }
```

Comment:
IS this after merging the split regions? (I don't see a `samtools merge` entry here)

Reply:
Ah it's further up, we should make sure these entries are in a consistent order following the workflow (ie., these _DEDUPPED entries should go before preseq (I guess this was derived from a merge conflict resolution)